christfellowshipchurch/web-app-v2
# Fixed Navbar Transparency Bug

### About
This PR fixes the transparent navbar after a user logs in. I added a separate `TransparentHeader` and a `useEffect` to help capture the navbar state better.

### Test Instructions
* run web locally
* login from homepage
* open homepage in new tab to make sure transparent navbar doesn't appear.
* continue logging in/out to test for bugs

### Screenshots
![image](https://user-images.githubusercontent.com/46049974/143948024-3c65bae8-7fcb-4eec-a892-ab514e0142b7.png)
![image](https://user-images.githubusercontent.com/46049974/143948205-25e0e99d-c108-4721-8c2a-b4e648655e56.png)

### Closes Tickets
[CFDP-1830]


[CFDP-1830]: https://christfellowshipchurch.atlassian.net/browse/CFDP-1830?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
